File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
n = int(input())
num_arr = list(map(int, input().split()))
min, max = num_arr[0], num_arr[0]
for num in num_arr[1:]:
    if num > max:
        max = num
    if num < min:
        min = num
print(min, max)

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main(void)
{
    int n, min, max;
    int arr[1000000];
    scanf("%d", &n);
    for (int i = 0; i < n; i++)
        scanf("%d", &arr[i]);
    min = arr[0];
    max = arr[0];
    for (int i = 1; i < n; i++)
    {
        if (arr[i] < min) min = arr[i];
        if (arr[i] > max) max = arr[i];
    }
    printf("%d %d", min, max);
    return 0;
}